HW2-kata4.py

```diff
@@ -0,0 +1,87 @@
+""" 4 kyu - Text align justify
+https://www.codewars.com/kata/537e18b6147aa838f600001b
+
+Your task in this Kata is to emulate text justification in monospace font.
+You will be given a single-lined text and the expected justification width.
+The longest word will never be greater than this width.
+
+Here are the rules:
+
+- Use spaces to fill in the gaps between words.
+- Each line should contain as many words as possible.
+- Use '\n' to separate lines.
+- Gap between words can't differ by more than one space.
+- Lines should end with a word not a space.
+- '\n' is not included in the length of a line.
+- Large gaps go first, then smaller ones ('Lorem--ipsum--dolor--sit-amet,'
+ (2, 2, 2, 1 spaces)).
+- Last line should not be justified, use only one space between words.
+- Last line should not contain '\n'
+- Strings with one word do not need gaps ('somelongword\n').
+
+Example with width=30:
+
+Lorem  ipsum  dolor  sit amet,
+consectetur  adipiscing  elit.
+Vestibulum    sagittis   dolor
+mauris,  at  elementum  ligula
+tempor  eget.  In quis rhoncus
+nunc,  at  aliquet orci. Fusce
+at   dolor   sit   amet  felis
+suscipit   tristique.   Nam  a
+imperdiet tellus.
+"""
+
+
+def justify(text, width):
```

Comment:
maybe it has sense to get a list of all fords in appropriate order `words = test.split()`
next you can take from it words that no more than width (with at least one space) and write function to form string for these words? 
it seems to me it would be shorter and clearer